.travis.yml

```diff
@@ -14,20 +14,6 @@ addons:
   chrome: stable
   apt:
     packages:
-      # I suspect that not all of these are necessary
```

Comment:
@manthey so these are now available on travis instances?

Reply:
xvfb is fully available and that is sufficient for Firefox and Chrome headless.